ocaml/xapi/pool_db_backup.ml

```diff
@@ -250,7 +250,7 @@ let fetch_database_backup ~master_address ~pool_secret ~force =
 (* Master sync thread *)
 let pool_db_backup_thread () = Debug.with_thread_named "pool_db_backup_thread" (fun () ->
     Server_helpers.exec_with_new_task "Pool DB sync" (fun __context ->
-        while (true) do
+        while (not !Xapi_globs.slave_dbs) do
```

Comment:
We should make `write_db_to_disk` from the local DB also conditional on this flag. I'd also prefer to not do `Debug.with_thread_named` and `Server_helpers.exec_with_new_task`, so leave this `while true`, and put an if-statement on top.